src/test/java/net/minecraftforge/debug/DeferredRegistryTest.java

```diff
@@ -57,7 +71,19 @@ public DeferredRegistryTest() {
         BLOCKS.register(modBus);
         ITEMS.register(modBus);
         CUSTOMS.register(modBus);
+        RECIPE_TYPES.register(modBus);
+        PLACED_FEATURES.register(modBus);
         modBus.addListener(this::gatherData);
+
+        MinecraftForge.EVENT_BUS.addListener(this::serverStarted);
+    }
+
+    public void serverStarted(ServerStartedEvent event) {
```

Comment:
Formatting: braces on newlines.
```suggestion
    public void serverStarted(ServerStartedEvent event)
    {
```